templates/gcp/fips.html

```diff
@@ -4,354 +4,446 @@
 {% block meta_description %}Ubuntu Pro FIPS for Google Cloud is an Ubuntu Pro image with FIPS certified modules enabled by default for FedRAMP, HIPAA, PCI and ISO compliance. Ubuntu Pro FIPS images are secure and optimised for production on public cloud.{% endblock %}
 {% block meta_copydoc %}https://docs.google.com/document/d/1Djxdzjy78bau8ELM4-MNGoM2YRa_QjpN1xJ4H4GjFyw/edit#heading=h.c6wjbvvp8mcm{% endblock meta_copydoc %}
 
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
 {% block content %}
-<section class="p-strip u-no-padding--top u-no-padding--bottom">
-  <div class="p-strip--suru-topped">
-    <div class="row u-equal-height">
-      <div class="col-7 col-medium-4">
+<section class="p-section--hero">
+    <div class="row--50-50">
+      <div class="col">
         <h1>Ubuntu Pro FIPS for Google Cloud compliance</h1>
-        <h2 class="p-heading--3">For Google Cloud compliance in production environments.</h2>
+        <h2 class="p-heading--5">For Google Cloud compliance in production environments.</h2>
         <p>Ubuntu Pro FIPS is the first and only FIPS 140-2 certified image for Google Cloud. Built upon the enhanced stability and security features of <a href="/pro">Ubuntu Pro</a>, Ubuntu FIPS is a critical foundation for federal programs and government contractors.</p>
       </div>
-      <div class="col-5 u-align--center u-vertically-center u-hide--medium u-hide--small">
-        {{
-          image(
-            url="https://assets.ubuntu.com/v1/6e176d9a-Google+Cloud+stacked.svg",
-            alt="",
-            width="250",
-            height="200",
-            hi_def=True,
-            loading="auto",
-          ) | safe
-        }}
+      <div class="col u-hide--small"> 
+        <div class="p-image-container--3-2 u-hide--small u-hide--medium">
+          <img class="p-image-container__image" src="https://assets.ubuntu.com/v1/b3b5e33e-google-cloud-hero.png" alt="" />
+        </div>
       </div>
     </div>
-  </div>
 </section>
 
-<section class="p-strip--light">
+<section class="p-section">
+  <div class="u-fixed-width p-section--shallow">
+    <hr />
+    <h2>What's in Ubuntu Pro FIPS?</h2>
+  </div>
   <div class="row">
-    <div class="col-12">
-      <h2>What's in Ubuntu Pro FIPS?</h2>
-      <ul class="p-matrix u-clearfix">
-        <li class="p-matrix__item">
-          <div class="p-matrix__content">
-            <h3 class="p-matrix__title">FIPS compliant by default</h3>
-            <div class="p-matrix__desc">
+    <div class="col-start-large-4 col-9 col-medium-6">
+      <hr class="p-rule--muted" />
+      <div class="row">
+        <div class="col-3 col-medium-2">
+          <div class="list-matrix">
+            <div class="list-matrix-container">
+              <p class="p-heading--5">FIPS compliant by default</p>
```

Comment:
There is too much space between the list title and paragraph items
<img width="294" alt="Screenshot 2024-09-03 at 12 12 51 PM" src="https://github.com/user-attachments/assets/f0da20d5-7006-478c-bbdf-fa0a92f523a0">

Reply:
For now we agreed with design to use this pattern until it is upstreamed in Vanilla